### PR TITLE
[FEAT]: Add/delete tasks functionality

### PR DESCRIPTION
**Pull request by shivanigangadharan**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,6 +13,8 @@ function App() {
         <Routes>
           <Route path="/" element={<Homepage />} />
           <Route path="/timerpage" element={<TimerPage />} />
+          <Route path="/settings" element={<Settings />} />
```
Comment (sharathnair9999) [inline src/App.js]: `/` can be omitted in route path for react router v6 except for home route

**Review by sharathnair9999 — APPROVED** [review]
Code is clean  good work

File: src/App.js (hunk 1 above)
Comment (shivanigangadharan) [reply src/App.js]: Done

File: src/components/Tasks/tasks.js [hunk 2]
```diff
@@ -10,10 +10,8 @@ export default function Tasks() {
     const short_uuid = uuid().slice(0, 8);
 
     const ToggleState = (ele) => {
```
Comment (spraveenofficial) [inline src/components/Tasks/tasks.js]: Naming could be better.

**Review by spraveenofficial — APPROVED** [review]

**Review by spraveenofficial — APPROVED** [review]
Good work .

File: src/components/Tasks/tasks.js (hunk 2 above)
Comment (shivanigangadharan) [reply src/components/Tasks/tasks.js]: changed it to `toggleTaskStatus`.

**Pull request merged**